schema/types/acname.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	valchars = `abcdefghijklmnopqrstuvwxyz0123456789.-/`
+	ACNameValChars = `abcdefghijklmnopqrstuvwxyz0123456789.-/`
```

Comment:
docstring I mean, not comment
(and the docstring for ACName should reference this)